src/Components/Address/useAddressAutocomplete.ts

```diff
@@ -38,6 +34,11 @@ export const useAddressAutocomplete = (
   ) => void
 } => {
   const [result, setResult] = useState<ProviderSuggestion[]>([])
+
+  // NOTE: Due to the format of this key (a long string of numbers that cannot be parsed as json)
+  // This key must be set in the env as a json string like SMARTY_EMBEDDED_KEY_JSON={ "key": "xxxxxxxxxxxxxxxxxx" }
+  const { key: apiKey } = getENV("SMARTY_EMBEDDED_KEY_JSON") || { key: "" }
```

Comment:
@erikdstock @TMcMeans I moved this line into this function so that I could write tests with and without the API key. When this line was at the top of the file, I was only able to mock the environment variable once per test file.